### PR TITLE
Fix deprecated cli command in the integration test

### PR DESCRIPTION
**Pull request by cw75**

### CODE REVIEW

**Comment by kenxu95** [comment]
ci

**Review by kenxu95 — APPROVED** [review]
I triggered an integration test run by commenting "ci" on this pull request, once that passes you can merge :)

**Comment by kenxu95** [comment]
Actually Fanjia's latest PR broke the tests. If the tests get past the server spinup part, we should be good.

**Comment by cw75** [comment]
@kenxu95 I don't think the integration test is picking up my change though.

**Comment by kenxu95** [comment]
urgh I hate hate hate github actions. Its impossible to test anything well. The comment-trigger seems to be broken.

Whatever, just merge it. Can you add the `integration-test.yml` file itself to the paths that would trigger a run on commit btw?

File: .github/workflows/integration-tests.yml [hunk 1]
```diff
@@ -69,7 +69,7 @@ jobs:
         run: pip3 install .
```
Comment (kenxu95) [inline .github/workflows/integration-tests.yml:69]: Can we add `.github/workflows/integration-test.yml` to the `paths` field at the beginning of this test

Comment (cw75) [reply .github/workflows/integration-tests.yml:70]: like this? see my last commit.

Comment (kenxu95) [reply .github/workflows/integration-tests.yml:70]: should be "integration-tests.yml"

**Pull request merged**